lib/health_cards/issuer.rb

```diff
@@ -1,41 +1,30 @@
 # frozen_string_literal: true
 
-require_relative 'keys'
+require_relative 'file_key_store'
+
+require 'json/jwk'
 
 module HealthCards
   # Verifiable Credential Issuer
   #
   # https://www.w3.org/TR/vc-data-model/#issuer
   class Issuer
-    include Keys
-
-    attr_reader :key_path
-
-    def initialize(key_path = Pathname.new('.'))
-      @key_path = key_path.is_a?(String) ? Pathname.new(key_path) : key_path
+    def initialize(key_store)
+      @key = key_store.load_or_create_key
     end
 
-    def signing_key_path
-      key_path.join 'signing_key.pem'
+    def signing_key
+      @signing_key ||= JSON::JWK.new(@key, use: 'sig', alg: 'ES256')
```

Comment:
The `json/jwt` gem defines `OpenSSL::PKey::EC#to_jwk` so I think you can remove the require and just do

```suggestion
      @signing_key ||= @key.to_jwk(use: 'sig', alg: 'ES256)
```

Reply:
OK, I'll incorporate this locally in my next push